include/alpaka/warp/WarpGenericSycl.hpp

```diff
@@ -34,21 +34,30 @@ namespace alpaka::warp::trait
     {
         static auto getSize(warp::WarpGenericSycl<TDim> const& warp) -> std::int32_t
         {
-            auto const sub_group = warp.m_item.get_sub_group();
+            auto const sub_group = warp.m_item_warp.get_sub_group();
             // SYCL sub-groups are always 1D
-            return static_cast<std::int32_t>(sub_group.get_local_linear_range());
+            return static_cast<std::int32_t>(sub_group.get_max_local_range()[0]);
         }
     };
 
     template<typename TDim>
     struct Activemask<warp::WarpGenericSycl<TDim>>
     {
+        // FIXME This should be std::uint64_t on AMD GCN architectures and on CPU,
+        // but the former is not targeted in alpaka and CPU case is not supported in SYCL yet.
+        // Restrict to warpSize <= 32 for now.
```

Comment:
This is not supported by LLVM at the moment, right? We discussed this in the alpaka VC today.